### PR TITLE
darsh175223/try course button

### LINKED ISSUE #6: Try Course Button
Does not need to bring us to anywhere yet

image

This will help
https://tailwindcss.com/docs/ring-width

### PR DESCRIPTION
**Pull request by darsh175223**
I created a new file in the components folder for the Try Course button. The colors and shapes of the button are in line with the figma design

### CODE REVIEW

**Comment by stanleylew5** [comment]
make sure you link your issue, also sync your github emails or use one account (whichever one was invited) for your issues

File: src/components/try-course-button.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const TryCourseButton = () => {
+  return (
+    <div className="relative inline-block w-[20%] p-1.5">
+      {/* Larger shape beneath */}
+      <div className="absolute inset-0 rounded-full bg-[#cff2aa]"></div>
+
+      {/* Main button */}
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion

```

File: src/components/try-course-button.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const TryCourseButton = () => {
+  return (
+    <div className="relative inline-block w-[20%] p-1.5">
+      {/* Larger shape beneath */}
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion

```

File: src/components/try-course-button.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <div className="absolute inset-0 rounded-full bg-[#cff2aa]"></div>
+
+      {/* Main button */}
+      <div className="flex justify-center">
+        <button className="relative w-[96%] rounded-full bg-[#eaf7dc] px-12 py-3 text-xl font-bold uppercase text-black shadow-md transition-all duration-300 ease-in-out hover:shadow-lg">
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion
        <button className="relative w-[96%] rounded-full bg-[#eaf7dc] px-12 py-3 text-xl font-bold text-black shadow-md">
```

Comment (stanleylew5) [reply src/components/try-course-button.tsx]: also, use dark mode colors since we're going to start with dark mode first. instead of hard coding #eaf7dc, add the color within tailwind config file. pull from dev first though `git pull origin`

File: src/components/try-course-button.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+          <span className="relative z-10">TRY COURSE</span>
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion
          <p>TRY COURSE</p>
```

File: src/components/try-course-button.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+          <div className="absolute inset-0 rounded-full bg-gradient-to-b from-transparent to-[#eaf7dc] opacity-50"></div>
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion
        
```
not quite sure what this is for it does nothing

File: src/components/try-course-button.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+const TryCourseButton = () => {
+  return (
+    <div className="relative inline-block w-[20%] p-1.5">
```
Comment (stanleylew5) [inline src/components/try-course-button.tsx]:
```suggestion
    <div className="relative w-[20%] p-1.5">
```

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
tldr, this code overcomplicates the issue it can be something similar to the below.obviously some styling needs to be done but it takes care of all the absolute inset positioning and all that you're going through right now. try following the code above this comment. and import your colors within the tailwind config file 

```
<div className="bg-green-500 rounded-full py-2 px-4">
        <button className="text-2xl bg-[#eaf7dc] rounded-full px-4 py-2 text-black">TRY COURSE</button>
</div>
```

**Comment by darsh175223** [comment]
I changed the colors to dark mode, removed all the tags that don't serve a purpose and changed the text and the widths to be dependent on the screen size(ex: text-[5vw]).
